### DISCUSSION TITLE
CPU FAST (PL2) not working

### DISCUSSION THREAD

**Discussion by hadixcanimm** [seerge/g-helper#1747] [Q&A] [answered]
I have a intel i5 10300h processor and with the 0.138.0 version update, the CPU FAST (PL2) setting came, but changing this setting does not apply any changes on my device.

![Ekran Alıntısı](https://github.com/seerge/g-helper/assets/114208757/977a03c5-d43a-484d-abb2-ec802d5928e8)

**Comment by seerge** [comment]
@hadixcanimm as I understand you didn't had any power sliders there before, correct ?

**Reply by hadixcanimm** [reply]
this slider was not here before. I use every new version, it came exclusive to this version.

**Reply by seerge** [reply, marked as answer]
@hadixcanimm right, in this build new slider should be hidden for you, as your system doesn't support that anyway :)

You can try Throttlestop app to limit power on a level of windows driver if you want tho 

[GHelper.zip](https://github.com/seerge/g-helper/files/13692086/GHelper.zip)
